config/settings.py

```
@@ -313,3 +317,5 @@
 # This is used to determine a users primary email and should be set to a comma separated list
 # e.g: 'mobile.ukti.gov.uk, digital.trade.gov.uk'
 DEFAULT_EMAIL_ORDER = env('DEFAULT_EMAIL_ORDER', default="")
+
+TRUSTED_IPS = env('TRUSTED_IPS', default='127.0.0.1')
```

Minor thing, but I suspect better to have no default, and raise an exception if TRUSTED_IPS is not set There is a bit of a gotcha: if it ends up determining IP by REMOTE_ADDR, which is often 127.0.0.1 in PaaS(?), then not setting it (e.g. accidentally) would mean every single request is trusted.

---

This is a fairly old PR - unsure if ti will be revived or not. If it is, it'll need a fair amount of reworking.